AppBuilder/platform/process/tasks/ABProcessGatewayExclusive.js

```diff
@@ -58,7 +58,7 @@ module.exports = class ABProcessGatewayExclusive extends (
 
          var connectedElement = this.process.elementForDiagramID(conn.to);
 
-         var DF = new RowFilter(App, `${id}_${conn.id}_filter`, this.AB);
+         var DF = new FilterComplex(App, `${id}_${conn.id}_filter`, this.AB);
```

Comment:
Since we have `this.AB`  lets standardize on using `var DF = this.AB.filterComplexNew("id");` for these calls.

Does `FilterComplex` actually use `App` anymore?  If not, this is a good time to start removing the `App` parameter, since it will eventually be removed from our platform code.